api/builtins/PatchStrategicMergeTransformer.go

```diff
@@ -75,7 +75,8 @@ func (p *PatchStrategicMergeTransformerPlugin) Transform(m resmap.ResMap) error
 			return err
 		}
 		if err = m.ApplySmPatch(
-			resource.MakeIdSet([]*resource.Resource{target}), patch); err != nil {
+			resource.MakeIdSet([]*resource.Resource{target}),
```

Comment:
This format looks weird, shouldn't there an indent before `path`?

Reply:
dropped the change